### ISSUE TITLE
[Suggestion] A player command to disable mcmmo skills and abilities

### ISSUE THREAD

**Issue by PeterDorvay** [mcMMO-Dev/mcMMO#4457]
My suggestion is a command to disable mcmmo per-player, similar to `/mcability` but also disables ALL passive skills such as armor impact and critical strike from axes. This command would ideally just completely disable mcmmo for the player that ran the command. The reason for this would be to allow for players who have extremely high mcmmo levels to fight players who have low mcmmo levels on an even ground.

Thanks!

**Comment by CodeGenStudio** [comment]
I agree with this, it would be amazing if there was a command to disable mcmmo completely (non-passive AND passive) as, I want to hold a PvP Event on the server I'm playing but there are issues since there are players with extremely high passive/non-passive skills, which would make the competition unfair.

Many thanks

**Comment by Zoro-6191** [comment]
hey, is there any update on this

**Comment by clivet268** [comment]
hey, is there any update on this
